plugins/GSdx/GLLoader.cpp

```diff
@@ -276,9 +276,9 @@ namespace GLLoader {
 			mesa_amd_buggy_driver = intel_buggy_driver = true;
 
 #ifdef _WIN32
-		buggy_sso_dual_src = intel_buggy_driver || fglrx_buggy_driver || legacy_fglrx_buggy_driver;
+		buggy_sso_dual_src = intel_buggy_driver || legacy_fglrx_buggy_driver;
```

Comment:
All drivers before 2016 were still blacklisted so resolution was around 5-6 months. Anyway, I updated the PR to blacklist driver from the start of 2016. If people found a version that doesn't work, they will need to either upgrade or downgrade their drivers. (Or use the stable PCSX2)